includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -24,26 +33,39 @@ public function getNavigation() {
 	 */
 	public function getAllElements() {
 		$params = $this->getRequestParameters();
+		$footerModel = new DesignSystemGlobalFooterModel(
+			$params[static::PARAM_PRODUCT],
+			$params[static::PARAM_ID],
+			$params[static::PARAM_LANG]
+		);
+		$navigationModel = new DesignSystemGlobalNavigationModel(
+			// TODO: Use `product` and `id` parameters?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)